packages/pds/src/api/com/atproto/label/queryLabels.ts

```diff
@@ -0,0 +1,13 @@
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+import { pipethrough } from '../../../../pipethrough'
+
+export default function (server: Server, ctx: AppContext) {
+  server.com.atproto.label.queryLabels({
+    auth: ctx.authVerifier.accessCheckTakedown,
+    handler: async ({ req, auth }) => {
+      const requester = auth.credentials.did
+      return pipethrough(ctx, req, requester)
+    },
+  })
+}
```

Comment:
Hey good news, we actually don't need to add every new endpoint to the pds anymore 😆 Daniel covered this in #2425, so now the PDS will proxy any lexrpc endpoint.